docs/src/pages/components/textfield/demo.tsx

```diff
@@ -76,67 +75,134 @@ export const TextFieldDemo = () => {
     [value],
     [variation],
   ] = textFieldProps;
+
+  const code =
+    `<TextField` +
+    (alignContent
+      ? `
```

Comment:
Non-blocking, and more of a thought exercise... I wonder how we can make these code generator blocks easier to read/write... I don't have an answer, but we should probably think about it as we make these updates.

Reply:
Yea, that's something we should think about, when it comes to a primitive with tons of props to demo